tests/test_basic_ops_dict.py

```diff
@@ -539,17 +539,39 @@ def test_set_with_invalid_key() -> None:
         cfg[1] = "a"  # type: ignore
 
 
-def test_set_anynode() -> None:
+@pytest.mark.parametrize("value", [1, 3.14, True, None, Enum1.FOO])  # type: ignore
```

Comment:
these tests should probably be in test_nodes.py

Reply:
Well this one tests if the node is not replaced. That cannot be done in test_nodes.

Reply:
But yeah, I missed adding theses test_cases in test_valid_inputs in test_nodes

Reply:
It's okay. Please move the tests to test_nodes.